x-pack/plugins/apm/public/components/app/Home/index.tsx

```diff
@@ -93,7 +109,11 @@ export function Home({ tab }: Props) {
         <EuiFlexGroup alignItems="center">
           <EuiFlexItem grow={false}>
             <EuiTitle size="l">
-              <h1>APM</h1>
+              <h1>
+                {selectedTab.name === 'rum-overview'
+                  ? EndUserExperienceLabel
+                  : 'APM'}
+              </h1>
```

Comment:
@shahzad31 @drewpost As mentioned in my design improvements PR, I think this is not the way to go about it. The Real User Monitoring view is still within APM (see breadcrumb), and we keep that title for all those views (Services, Traces, and Service maps). I find replacing the main title confusing.